prow_config.yaml

```diff
@@ -3,3 +3,14 @@
 workflows:
```

Comment:
I guess my previous question was, how can I use `prow_config.json` in pipelines repo?